src/test/scala/com/cognite/spark/datasource/BaseUrlTest.scala

```diff
@@ -6,19 +6,19 @@ class BaseUrlTest extends FlatSpec with SparkTest {
   private val greenfieldApiKey = System.getenv("TEST_API_KEY_GREENFIELD")
   private val readApiKey = System.getenv("TEST_API_KEY_READ")
 
-  it should "read different files metadata from greenfield and api" taggedAs GreenfieldTest in {
+  it should "read different files metadata from greenfield and api" taggedAs GreenfieldTest ignore {
 
     val dfGreenfield = spark.read
       .format("com.cognite.spark.datasource")
       .option("apiKey", greenfieldApiKey)
-      .option("type", "files")
+      .option("type", "assets")
```

Comment:
I guess it doesn't matter too much but we picked files because it has fewer items than assets. Using assets will probably be a bit slower.

Reply:
Agreed, changed this since the sdk couldn't read from Greenfield, so it broke the test. Can change it back once we have that in place.